crates/flake8_to_ruff/pyproject.toml

```diff
@@ -26,7 +26,7 @@ requires-python = ">=3.7"
 repository = "https://github.com/charliermarsh/ruff#subdirectory=crates/flake8_to_ruff"
 
 [build-system]
-requires = ["maturin>=0.14,<0.15"]
+requires = ["maturin>=0.15.1,<0.16"]
```

Comment:
I am personally against max version constraints. See this blogpost for good arguments about why they are not useful, and actually detrimental: 
https://iscinumpy.dev/post/bound-version-constraints/

Reply:
Note that this is not a dependency in the classical sense, but this is the build system. Without upper bounding this, packages published with previous maturin version would become unbuildable once we publish a maturin version with breaking changes